scripts/gcp_caper_server/create_instance.sh

```diff
@@ -0,0 +1,314 @@
+#!/bin/bash
+set -e
```

Comment:
Would be good to be more strict:
```suggestion
set -euo pipefail
```

Reply:
Done in other commits as you suggested.